mysite/templates/psa/modal.html

```diff
@@ -27,7 +27,7 @@ <h4 class="modal-title">Re-send activation link</h4>
   </div>
 
 <div id="email-modal" class="modal fade">
-  <form action="{% url "social:complete" "email" %}" method="post" role="form">{% csrf_token %}
+  <form action="{% url "social:complete" "email" %}?next={{ next }}" method="post" role="form">{% csrf_token %}
```

Comment:
@cjlee112 This addition is caused by python-social-auth requirements to handle `next` param. I added this according to python-social-auth documentation.
And yes, I also not happy with this mixing.